README.md

```diff
@@ -1,27 +1,34 @@
 # Role Name
 
-A brief description of the role goes here.
+This role installs and configures a redis server. It allows to set all configuration variables supported by redis.
 
 
 ## Requirements
 
-Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here.
-For instance, if the role uses the EC2 module or depends on other Ansible roles, it may be a good idea to mention in this section that the boto package is required.
+An dpkg- or pacman-based Linux distribution.
```

Comment:
I was really happy when I thought you had pacman in mind.... 😞

Reply:
I'm sure he just forgot the pacman tasks ;)

Reply:
added pacman tasks